package.json

```diff
@@ -107,7 +107,7 @@
     "@elastic/datemath": "link:bazel-bin/packages/elastic-datemath",
     "@elastic/elasticsearch": "npm:@elastic/elasticsearch-canary@8.1.0-canary.2",
     "@elastic/ems-client": "8.0.0",
-    "@elastic/eui": "46.1.0",
+    "@elastic/eui": "46.1.1",
```

Comment:
Probably need to commit the `yarn.lock` changes, also

Reply:
haha woops 🤦